content/en/docs/tasks/job/indexed-parallel-processing-static.md

```diff
@@ -0,0 +1,184 @@
+
+<!-- overview -->
+
+
+In this example, you will run a Kubernetes Job that uses multiple parallel
+worker processes.
+Each worker is a different container running in its own Pod. The Pods have an
+_index number_ that the control plane sets automatically, which allows each Pod
+to identify which part of the overall task to work on.
+
+The pod index is available in the {{< glossary_tooltip text="annotation" term_id="annotation" >}}
+`batch.kubernetes.io/job-completion-index` as string representing its
+decimal value. In order for the containerized task process to obtain this index,
+you can publish the value of the annotation using the [downward API](/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#the-downward-api)
+mechanism.
+For convenience, the control plane automatically sets the downward API to
+expose the index in the `JOB_COMPLETION_INDEX` environment variable.
+
+Here is an overview of the steps in this example:
+
+1. **Create an image that can read the pod index**. You might modify the worker
+   program or add a script wrapper.
+2. **Start an Indexed Job**. The downward API allows you to pass the annotation
+   as an environment variable or file to the container.
+
+## {{% heading "prerequisites" %}}
+
+Be familiar with the basic,
+non-parallel, use of [Job](/docs/concepts/workloads/controllers/job/).
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+To be able to create Indexed Jobs, make sure to enable the `IndexedJob`
+[feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
+on the [API server](docs/reference/command-line-tools-reference/kube-apiserver/)
+and the [controller manager](/docs/reference/command-line-tools-reference/kube-controller-manager/).
+
+<!-- steps -->
+
+## Create a container image
+
+To access the work item from the worker program, you have a few options:
+
+1. Read the `JOB_COMPLETION_INDEX` environment variable. The Job
+   {{< glossary_tooltip text="controller" term_id="controller" >}}
+   automatically links this variable to the annotation containing the completion
+   index.
+1. Read a file that contains the completion index.
+1. Assuming that you can't modify the program, you can wrap it with a script
+   that reads the index using any of the methods above and converts it into
+   something that the program can use as input.
+  
+For this example, imagine that you chose option 3 and you want to run the
+[rev](https://man7.org/linux/man-pages/man1/rev.1.html) utility. This
+program accepts a file as an argument and prints its content reversed.
+
+```shell
+rev data.txt
+```
+
+This program is available in the [busybox container image](https://hub.docker.com/_/busybox):
+
+{{< tabs name="busybox container image" >}}
+{{{< tab name="Docker Hub" >}}
+docker.io/library/busybox
+{{< /tab >}}
+{{< tab name="Google Container Registry" >}}
+mirror.gcr.io/library/busybox
+{{< /tab >}}}
+{{< /tabs >}}
```

Comment:
As we're not pushing images, my earlier feedback doesn't apply and this can simplify to:
```suggestion
For this example, you'll use the `rev` tool from the [`busybox`](https://hub.docker.com/_/busybox) container image.
```

Reply:
Done